### PR TITLE
feat: Translate `commit_messages`

### PR DESCRIPTION
**Pull request by altankurt**
Translation of "commit_mesajları.md" in [git/temel_git/](https://github.com/kamp-us/turkce-odin-project/tree/main/temel_bilgiler/git/temel_git) directory is complete. ✌️

@cansirin @ubsoydan

### CODE REVIEW

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Artık iyi bir commit mesajı oluşturmanın sırrını öğrendiğimize göre, daha önceki commit mesajını düzeltmeye çalışalım:
+
+```git
+Şirket logosuna eksik bağlantıyı ve alternatif metni ekleyin.
+
+Ekran okuyucular, bu bilgi olmadan engelli kullanıcılara görüntüleri okumayacaktır
+```
+
+Ahh, bu daha iyi! :) Şimdi, geliştiriciler bu commit mesajını daha iyi anlayabilirler çünkü aşağıdakileri yapar:
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
Ahh, bu daha iyi! :) Şimdi, geliştiriciler bu commit mesajını daha iyi anlayabilirler çünkü bu commit mesajı aşağıdakileri sağlar:
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+- Kodunuzun eylemini belirten bir başlık sağlar (örneğin, "Şirket logosuna eksik bağlantı ve alternatif metin ekleme")
+- Commit'ın neden yapılması gerektiğine dair kısa ama net bir açıklama sağlayan bir gövde içerir (örneğin, "Ekran okuyucular, bu bilgi olmadan görüntüleri engelli kullanıcılara okumayacaktır")
+- Konuyu gövdeden yeni/boş bir satırla ayırır. Bu, takip etmenizi şiddetle tavsiye ettiğimiz bir en iyi uygulamadır. Diğer geliştiricilerin commit mesajlarını okumasını kolaylaştırır.
+
+### Ne zaman commit yapılmalıdır
+
+Bir commit'i düşünmenin iyi bir yolu, kodunuzun yapıldığı andaki "anlık görüntüsü" gibidir. Kodunuzun o noktaya kadar olan versiyonu, geri dönmeniz veya tekrar bakmanız için kaydedilecektir.
+
+Kod yazarken, kodda anlamlı bir değişiklik yaptığınızda her seferinde commit etmek en iyi uygulama olarak kabul edilir. Bu, ilerlemenizin bir zaman çizelgesini oluşturacak ve bitmiş kodunuzun bir anda ortaya çıkmadığını gösterecektir.
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
Kod yazarken, kodda anlamlı bir değişiklik yaptığınızda her seferinde değişiklikleri commitlemek en iyi uygulama olarak kabul edilir. Bu, ilerlemenizin bir zaman çizelgesini oluşturacak ve bitmiş kodunuzun bir anda ortaya çıkmadığını gösterecektir.
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Başka bir deyişle, üzerinde çalıştığınız bir kod parçasının istediğiniz gibi çalışmasını sağladığınızda, bir yazım hatasını düzelttiğinizde veya bir hatayı giderdiğinizde bir commit yapın.Deneyim kazandıkça, hangi değişikliklerin commit edilmesi gerektiği konusunda daha iyi bir anlayış geliştireceksiniz!
+
+Bir projede çalışırken, SONUNDA bir şeyi tam olarak düzgün hale getirdiğiniz bir zaman gelecek (bu, commit yapmanın iyi bir zamanı olacaktır) ve ardından belki 30 saniye ile birkaç gün sonra bozulacaktır. Neyi değiştirdiğiniz hakkında hiçbir fikriniz yok, her şey aynı _görünüyor_ ve o satırı düzenlediğinizi hatırlamıyorsunuz, ama ne yazık ki artık istediğiniz gibi çalışmıyor. Commit geçmişinizi geriye doğru kontrol edebilecek ya da kodunuzu ilk başta o parçayı çalışır hale getirdiğinizde yaptığınız son commit'e geri dönebileceksiniz veya kodunuzun o zaman neye benzediğini görmek için geri gidebileceksiniz.
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
Bir projede çalışırken, SONUNDA bir şeyi tam olarak düzgün hale getirdiğiniz bir zaman gelecek (bu, değişikliklerinizi commitlemek iyi bir zaman olacaktır) ve ardından belki 30 saniye ile birkaç gün sonra bozulacaktır. Neyi değiştirdiğiniz hakkında hiçbir fikriniz yok, her şey aynı _görünüyor_ ve o satırı düzenlediğinizi hatırlamıyorsunuz, ama ne yazık ki artık istediğiniz gibi çalışmıyor. Commit geçmişinizi geriye doğru kontrol edebilecek ya da kodunuzu ilk başta o parçayı çalışır hale getirdiğinizde yaptığınız son commite geri dönebileceksiniz veya kodunuzun o zaman neye benzediğini görmek için geri gidebileceksiniz.
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+1.  Bu makale, [How to Write a Git Commit Message](https://cbea.ms/git-commit), iyi commit mesajlarının nasıl yazılacağına dair tüm ana temelleri kapsamaktadır. Makalenin tamamı harika ve bilgilendirici ancak asıl önemli kısmı “The seven rules of a great commit message.”
+
+</div>
+
+### İpuçları ve hatırlanması gerekenler:
+
+- VSCode'u metin düzenleyiciniz olarak kullanmak (bunu Git Temelleri bölümünde ayarlamış olmalısınız), çok satırlı commit mesajlarını kolayca oluşturmanıza, her satırın karakter uzunluğunu kolayca görmenize ve yazımınızın doğru olduğundan emin olmak için [VSCode spell check extensions](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) kullanmanıza olanak tanır.
+- Aktif bir dil kullanın: "Kart üreticiyi düzelt"
+- "Kaydedildi" veya "güncellendi" gibi belirsiz commit mesajlarından kaçının
+- Erken ve sık sık commit yapın!
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
- Değişikliklerinizi erken ve sık sık commitleyin!
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+```git
+Şirket logosuna eksik bağlantıyı ve alternatif metni ekleyin.
+
+Ekran okuyucular, bu bilgi olmadan engelli kullanıcılara görüntüleri okumayacaktır
+```
+
+Ahh, bu daha iyi! :) Şimdi, geliştiriciler bu commit mesajını daha iyi anlayabilirler çünkü aşağıdakileri yapar:
+
+- Kodunuzun eylemini belirten bir başlık sağlar (örneğin, "Şirket logosuna eksik bağlantı ve alternatif metin ekleme")
+- Commit'ın neden yapılması gerektiğine dair kısa ama net bir açıklama sağlayan bir gövde içerir (örneğin, "Ekran okuyucular, bu bilgi olmadan görüntüleri engelli kullanıcılara okumayacaktır")
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
- Commitin neden yapılması gerektiğine dair kısa ama net bir açıklama sağlayan bir gövde içerir (örneğin, "Ekran okuyucular, bu bilgi olmadan görüntüleri engelli kullanıcılara okumayacaktır")
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+```
+Kod tabanında yaptığım değişiklik.
+```
+
+#### Gövde
+
+Ne yaptığınıza dair kısa ama net bir açıklama.
+
+```
+Commit'inizin hangi sorunu nasıl çözdüğünü açıklayın.
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
Commitinizin hangi sorunu nasıl çözdüğünü açıklayın.
```

File: temel_bilgiler/git/temel_git/commit_mesajları.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+Ahh, bu daha iyi! :) Şimdi, geliştiriciler bu commit mesajını daha iyi anlayabilirler çünkü aşağıdakileri yapar:
+
+- Kodunuzun eylemini belirten bir başlık sağlar (örneğin, "Şirket logosuna eksik bağlantı ve alternatif metin ekleme")
+- Commit'ın neden yapılması gerektiğine dair kısa ama net bir açıklama sağlayan bir gövde içerir (örneğin, "Ekran okuyucular, bu bilgi olmadan görüntüleri engelli kullanıcılara okumayacaktır")
+- Konuyu gövdeden yeni/boş bir satırla ayırır. Bu, takip etmenizi şiddetle tavsiye ettiğimiz bir en iyi uygulamadır. Diğer geliştiricilerin commit mesajlarını okumasını kolaylaştırır.
+
+### Ne zaman commit yapılmalıdır
+
+Bir commit'i düşünmenin iyi bir yolu, kodunuzun yapıldığı andaki "anlık görüntüsü" gibidir. Kodunuzun o noktaya kadar olan versiyonu, geri dönmeniz veya tekrar bakmanız için kaydedilecektir.
```
Comment (cansirin) [inline temel_bilgiler/git/temel_git/commit_mesajları.md]:
```suggestion
Bir commiti düşünmenin iyi bir yolu, kodunuzun yapıldığı andaki "anlık görüntüsü" gibidir. Kodunuzun o noktaya kadar olan versiyonu, geri dönmeniz veya tekrar bakmanız için kaydedilecektir.
```

**Comment by altankurt** [comment]
Thanks!✌️ @cansirin

**Review by cansirin — APPROVED** [review]
Perfecto, thank you so much 🚀 💯

**Pull request merged**